src/MICore/LaunchOptions.cs

```diff
@@ -273,6 +302,25 @@ public override string ExePath
         /// [Optional] Log strings written to stderr and examine for server started pattern
         /// </summary>
         public int ServerLaunchTimeout { get; private set; }
+
+        /// <summary>
+        /// [Optional] Path to a core dump file for the specified executable.
+        /// </summary>
+        public string CoreDumpPath
+        {
+            get
+            {
+                return this.coreDumpPath;
+            }
+            private set
+            {
+                // CoreDumpPath is allowed to be null/empty
+                if (!String.IsNullOrEmpty(value) && !LocalLaunchOptions.CheckPath(value))
```

Comment:
The core file needs to be on the machine where gdb is executing, which is not necessarily the local machine.